app/src/main/java/com/example/parkingsystemkotlin/mvp/view/base/ActivityView.kt

```diff
@@ -0,0 +1,20 @@
+package com.example.parkingsystemkotlin.mvp.view.base
+
+import android.app.Activity
+import android.content.Context
+import androidx.appcompat.app.AppCompatActivity
+import androidx.fragment.app.FragmentManager
+import java.lang.ref.WeakReference
+
+open class ActivityView(activity: Activity) {
+    private var activityRef: WeakReference<Activity> = WeakReference(activity)
+
+    private val activity: Activity?
+        get() = activityRef.get()
+
+    val context: Context?
+        get() = activity
```

Comment:
this should be `activity?.context`

Reply:
we will dismiss this comment based on what we just talked